### ISSUE TITLE
Async error in server's console

### ISSUE THREAD

**Issue by DaDlugosch** [ComfortablyCoding/strapi-plugin-publisher#69]
Hi, when I have started using this plugin, I encountered these error messages in server's console. It always happens every minute while application is running. Could you please look at it? I'm overloading my Sentry dashboard with these weird errors. However, in Strapi administration everything looks correct. Even publishing works as expected.

```
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
TypeError: Cannot read properties of null (reading 'publishedAt')
    at Object.toggle (/Users/test-user/Sites/rkckk/web.com/node_modules/strapi-plugin-publisher/server/services/publication-service.js:61:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Thanks.

**Comment by ComfortablyCoding** [comment]
That should only occur if the records don't exist which is not expected to happen. I will add a guard for it but strange that it is occuring.

**Comment by ComfortablyCoding** [comment]
This should be fixed in v1.3.4 (latest release)
